attest/certification.go

```diff
@@ -62,6 +64,20 @@ type VerifyOpts struct {
 	Hash crypto.Hash
 }
 
+// ActivateOpts specifies options for the key certification's challenge generation.
+type ActivateOpts struct {
+	// EK, the endorsement key, describes an asymmetric key whose
+	// private key is permanently bound to the TPM.
+	//
+	// Activation will verify that the provided EK is held on the same
+	// TPM as the key we're certifying. However, it is the caller's responsibility to
+	// ensure the EK they provide corresponds to the the device which
+	// they are trying to associate the certified key with.
+	EK crypto.PublicKey
+	// Hash is the hash structure used as
```

Comment:
Can you complete the comment here?

Is this required? How does someone compute this value? Does this imply ActivateOpts only supports tpm2?

Reply:
Renamed this value to be more descriptive. This value is the name digest of the key we're using to verify certification of the target key being activated. I've added a comment to explain how to derive this value and added a helper to construct an ActivateOpts struct

Reply:
> Does this imply ActivateOpts only supports tpm2?

That is correct, [though based on this comment on the existing CertificationParameters](https://github.com/google/go-attestation/blob/master/attest/certification.go#L40), this file currently only supports tpm2